src/components/fieldinfo/fieldinfo.scss

```diff
@@ -16,6 +16,16 @@ $any-hover-color: #666;
   user-select: none;
   overflow: hidden;
 
+  .remove {
```

Comment:
Can you remove this commit (ba503da51dabc1e17826a7bf5f5cc0f40f304df9) entirely? 

I don't think this PR address the problem is a correct way.  It introduces so many problems.

- Position absolute isn't the right solution for flex box
- max-width: 100px is way too small.  

If you want to address this problem, file an issue and address it in a separate PR.  (It's not the crux of this PR.)